core/src/entities/sboms/sbom.rs

```diff
@@ -0,0 +1,292 @@
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+use std::fmt::{Display, Formatter};
+use tracing::log::debug;
+
+use crate::entities::cyclonedx::{Bom, Component};
+use crate::entities::packages::{Dependency, Package};
+use crate::entities::sboms::SbomProviderKind;
+use crate::entities::scans::ScanRef;
+use crate::entities::xrefs::Xref;
+use crate::Error;
+
+/// An SBOM is a snapshot inventory of the components that make up a piece of software at a moment in time.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+#[skip_serializing_none]
+pub struct Sbom {
+    /// The unique identifier for the Sbom.
+    pub id: String,
+
+    /// The package manager for the [Sbom].
+    pub package_manager: Option<String>,
+
+    /// The Package URL for the package that the Sbom was generated from.
+    pub purl: Option<String>,
+
+    /// The instance of the [Sbom]. Forward-only incrementing counter.
+    pub instance: u32,
```

Comment:
@qtpeters  @sbolel  @jonshattuck  I really dislike this name `instance`.  Any suggestions for a better name?